src/requirements/data/index.ts

```diff
@@ -377,6 +378,11 @@ const json: RequirementsJson = {
     },
   },
   grad: {
+    MENGCS: {
+      name: 'Master of Engineering (MEng) in Computer Science',
```

Comment:
Any preference between `Master of Engineering (MEng) in Computer Science` vs `Master of Engineering (MEng) Program in Computer Science`? Or another option?

Reply:
What about just CS M.Eng to save some space